tests/zfs-tests/tests/functional/cli_root/zfs_diff/cleanup.ksh

```diff
@@ -0,0 +1,29 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
```

Comment:
I think you've got an old version of the CDDL here. You probably want to copy the one in tests/zfs-tests/include/math.shlib for example.

Reply:
This may very well be, according to my shell history i started from "zfs_inherit" files.